### ISSUE TITLE
couldn't install 

### ISSUE THREAD

**Issue by praneshp1org** [thiagoojack/packettracer-fedora#27]
[pranesh@fedora packettracer-fedora]$ sudo bash install.sh

The Packet Tracer installer was found at:
/home/pranesh/.local/share/Trash/files/CiscoPacketTracer_821_Ubuntu_64bit.deb
/home/pranesh/Downloads/CiscoPacketTracer_821_Ubuntu_64bit.deb

Removing old version of Packet Tracer from /opt/pt
Installing dependencies
Last metadata expiration check: 0:14:45 ago on Wed 21 Jun 2023 09:29:03 PM +0545.
Package binutils-2.39-9.fc38.x86_64 is already installed.
Package qt5-qtmultimedia-5.15.10-1.fc38.x86_64 is already installed.
Package qt5-qtwebengine-5.15.12-6.fc38.x86_64 is already installed.
Package qt5-qtnetworkauth-5.15.10-1.fc38.x86_64 is already installed.
Package qt5-qtwebsockets-5.15.10-1.fc38.x86_64 is already installed.
Package qt5-qtwebchannel-5.15.10-1.fc38.x86_64 is already installed.
Package qt5-qtscript-5.15.10-1.fc38.x86_64 is already installed.
Package qt5-qtlocation-5.15.10-1.fc38.x86_64 is already installed.
Package qt5-qtsvg-5.15.10-1.fc38.x86_64 is already installed.
Package qt5-qtspeech-5.15.10-1.fc38.x86_64 is already installed.
Dependencies resolved.
Nothing to do.
Complete!
Extracting files
no entry /home/pranesh/Downloads/CiscoPacketTracer_821_Ubuntu_64bit.deb in archive
tar: packettracer/control.tar.xz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
tar: packettracer/data.tar.xz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
cp: cannot stat 'packettracer/usr': No such file or directory
cp: cannot stat 'packettracer/opt': No such file or directory
sed: can't read /usr/share/applications/cisco-pt.desktop: No such file or directory
sed: can't read ./packettracer/postinst: No such file or directory
sed: can't read ./packettracer/postinst: No such file or directory
sudo: ./packettracer/postinst: command not found
[pranesh@fedora packettracer-fedora]$

**Comment by thiagoojack** [comment]
I tried with two installers on different directories and reproduce this issue. Try to remove one of the installer and run script again.
I tried to install Packet Tracer with this script with only one installer, and have no problem. Watch this video if you have any doubts:
[Packet Tracer 8.2.1 Fedora 38 installation via script](https://www.youtube.com/watch?v=g9TITCwhOxI)

I will resolve this problem soon.
